include/bluetooth/radio_notification_cb.h

```diff
@@ -69,7 +69,7 @@ struct bt_radio_notification_conn_cb {
  * That gives a worst case combined clock accurary of 1000 ppm.
  * This results in 1 ms drift per second.
 
- * See Bluetooth Core_v5.4, Vol 6, Part 4.2.4 for more details on clock drift.
+ * See Bluetooth Core Specification Vol 6, Part 4.2.4 for more details on clock drift.
```

Comment:
Part and Section are messed up -- is this referring to Part B, Section 4.2.4?
```suggestion
 * See Bluetooth Core Specification, Vol 6, Part B, Section 4.2.4, for more details on clock drift.
```